### PR TITLE
With finger exercises as a starting point, give pointers on asking for help

### PR DESCRIPTION
**Pull request by lamagnifica**
@catong @srpearce @pdesjardins this is where I found myself when I started thinking about how to describe what to do when you encounter a term like “finger exercises” in your edx.org course…

### CODE REVIEW

File: en_us/shared/students/SFD_introduction.rst [hunk 1]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+
+  ******************
+  Learning in a MOOC
+  ******************
+
+  A massive, open, online course (MOOC) typically uses videos and text to
+  present course content. Other content, such as textbooks, articles, and other
+  reading assignments, might also be included. Assessments, in the form of
+  practice questions, homework assignments, and exams, are an important part of
+  the learning experience. Discussion topics help learners share information
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: I'd suggest turning some of these sentences around so that they are more from the learner's point of view, even though I think here you're listing types of course content. "You can participate in discussions to share information and collaborate with other learners and the course team." "To see and try out different types of course content, enroll in the edX Demo course."

File: en_us/shared/students/SFD_introduction.rst [hunk 2]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  and collaborate with each other and the course team. If you want to practice,
+  you can enroll in the `edX Demo course`_.
+
+  These basic components are present in virtually every edX course. In
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: Starting a new paragraph with "These" was confusing, since the components referred to were in the previous paragraph. Recommend rephrasing. Maybe: "In addition to the basic components described previously, courses can include enhanced components, such as..."

File: en_us/shared/students/SFD_introduction.rst [hunk 3]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  addition, the faculty and other course team members can design their courses
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: Seems unnecessarily distracting to bring in "faculty and other course team members". See suggestion in previous comment.

File: en_us/shared/students/SFD_introduction.rst [hunk 4]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  live all over the world, and you bring a wide variety of educational
+  backgrounds, and educational goals, to the edX courses that you take. Part of
+  the MOOC learning experience that everyone shares is encountering new ways of
+  learning. As a result, you will sometimes have questions about what is
+  expected, how to do something, or what a word or phrase means.
+
+  If you have a question about something you encounter in a course, try these
+  options.
+
+  * Check the **Home** page. Course teams post frequent updates to this page,
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: "frequent" might not always be true, so I'd delete "frequent". Suggest emphasizing that the Home page is where course teams communicate with them vs highlighting frequency. For examples of Home page updates, include something that ties back to the reason you're mentioning it here: course team updates might include explanations of exercise or assessment types that are included in the course. That would be a more valuable example than due dates, upcoming milestones, etc.

File: en_us/shared/students/SFD_introduction.rst [hunk 5]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    which can include reminders about when graded assignments are due, or
+    announce upcoming events or milestones.
+
+  * Use the :ref:`course discussions<Explore Posts>`. If you encounter an
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: "Use the course discussions" -> "Participate in course discussions."

File: en_us/shared/students/SFD_introduction.rst [hunk 6]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    unfamiliar word or phrase, such as "finger exercise" or "board work",
+    search for it on the **Discussion** page, or post a question about it
+    yourself. Your questions give the course team useful feedback about how
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: Suggest adding additional sentence to expand on the encourage to post and tie back to the original reason. "Other learners might be able to answer your question, or might have the same question themselves. Your comments and questions give the course team useful feedback for improving the course."

File: en_us/shared/students/SFD_introduction.rst [hunk 7]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    they might be able to improve the course.
+
+  * Investigate other course-specific resources. If there is a course
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: After the first example you talk about something outside the course. I'd delete "course-specific". 
"Investigate other resources. Within your course, if a course wiki exists, it can be a good source of information. Outside your course, ..." (Maybe chat tool as one of the examples in addition to FB and Twitter?)
I'd delete the unnecessary commas after "asking" and "answering" - sounds a bit pedantic.

File: en_us/shared/students/SFD_introduction.rst [hunk 8]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    :ref:`wiki<SFD Wiki>` on the **Wiki** page, additional resources might be
+    listed there. Outside the course, there might be a Facebook page or a
+    Twitter feed where other leareners are asking, and answering,
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: typo in "leareners"

File: en_us/shared/students/SFD_introduction.rst [hunk 9]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  to include enhanced components, from team problem-solving exercises and lab
+  work to live demonstrations and ask me anything (AMA) sessions. Each edX
+  course has a unique design, and course teams are continuously looking for new
+  ways to engage learners and teach more effectively.
+
+  The learners in edX courses are an amazingly diverse group of people. You
+  live all over the world, and you bring a wide variety of educational
+  backgrounds, and educational goals, to the edX courses that you take. Part of
+  the MOOC learning experience that everyone shares is encountering new ways of
+  learning. As a result, you will sometimes have questions about what is
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: "will sometimes" - > "might"?

File: en_us/shared/students/SFD_introduction.rst [hunk 10]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  you can enroll in the `edX Demo course`_.
+
+  These basic components are present in virtually every edX course. In
+  addition, the faculty and other course team members can design their courses
+  to include enhanced components, from team problem-solving exercises and lab
+  work to live demonstrations and ask me anything (AMA) sessions. Each edX
+  course has a unique design, and course teams are continuously looking for new
+  ways to engage learners and teach more effectively.
+
+  The learners in edX courses are an amazingly diverse group of people. You
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: Can't quite put my finger on why the first sentence by itself doesn't work for me. "Amazingly diverse" seems awkward. A different sentence split maybe? "Learners in edX courses are a diverse group of people who live all over the world. You bring a wide variety..."

File: en_us/shared/students/SFD_introduction.rst [hunk 11]
```diff
@@ -8,13 +8,62 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  live all over the world, and you bring a wide variety of educational
+  backgrounds, and educational goals, to the edX courses that you take. Part of
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: "Regardless of your background or your reasons for taking an edX course, a part of the MOOC learning experience that everyone shares is encountering new ways of learning."

**Comment by catong** [comment]
@lamagnifica I think this topic is a good addition and will be helpful. There is flow from paragraph to paragraph, so I don't see the need to add subheads. In general, I think the language could be more simple and direct in addressing learners, and possibly slightly more casual in tone. Suggestions in comments.

File: en_us/shared/students/SFD_introduction.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: right. I was originally thinking bulleted list, with the key words (assessments, discussions) first, but then went back to a paragraph...

File: en_us/shared/students/SFD_introduction.rst (hunk 7 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: I was thinking in terms of a course-specific Facebook page or course-specific Twitter feed, rather than generally. Will tune to make that clearer.

Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: @catong do we have a chat tool that teams can integrate into courses?

Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: I left that comment while under the impression that you were talking about non-integrated pages and channels, although I do recall mention of courses using various other discussion or forums because of the limitations of our forums. These weren't necessarily integrated, but potentially something like a Slack channel that course teams created.

Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: true, though we no longer let them do that -- the Advanced setting to replace the disussion forum is deprecated now

File: en_us/shared/students/SFD_introduction.rst [hunk 12]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
 
+.. only:: Partners
+
+  ******************
+  Learning in a MOOC
+  ******************
+
+  A massive, open, online course (MOOC) typically uses videos and text to
+  present course content, although you might also encounter other types of
+  reading assignments, such as textbooks and online articles. You will find
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: I think the seconds sentence should make it clearer that you're describing different elements of a MOOC. I read far into it, still thinking it was about presenting course content. Maybe something like:

A MOOC also includes assessments, in the form of practice questions, homework assignments, and exams.

File: en_us/shared/students/SFD_introduction.rst [hunk 13]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  that the assessments, in the form of practice questions, homework
+  assignments, and exams, are also an important part of the learning experience
+  in a MOOC. To help you share information, ask questions, and collaborate with
+  other learners and the course team, course discussions are available.
+
+  Videos and other types of course content, assessments, and discussions are
+  used in virtually every edX course. If you want to review examples of these
+  components and practice using them, you can enroll in the `edX Demo course`_.
+
+  Depending on the course, other, enhanced components might also be included,
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: The commas read strangely for me. Maybe get rid of "other"?

File: en_us/shared/students/SFD_introduction.rst [hunk 14]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  A massive, open, online course (MOOC) typically uses videos and text to
+  present course content, although you might also encounter other types of
+  reading assignments, such as textbooks and online articles. You will find
+  that the assessments, in the form of practice questions, homework
+  assignments, and exams, are also an important part of the learning experience
+  in a MOOC. To help you share information, ask questions, and collaborate with
+  other learners and the course team, course discussions are available.
+
+  Videos and other types of course content, assessments, and discussions are
+  used in virtually every edX course. If you want to review examples of these
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: This is the first instance of "edX" in the section. Would "MOOC" work better? I'm thinking about how this will read for an Open edX learner.

File: en_us/shared/students/SFD_introduction.rst [hunk 15]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  components and practice using them, you can enroll in the `edX Demo course`_.
+
+  Depending on the course, other, enhanced components might also be included,
+  from team problem-solving exercises and lab work to live demonstrations and
+  ask me anything (AMA) sessions. Each edX course has a unique design, and
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: "edX course" or "MOOC"? I think maybe MOOC here, plus a global examination of what this section calls the courses it's about.

File: en_us/shared/students/SFD_introduction.rst (hunk 14 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: @pdesjardins I have this set up to be included in the edX guide only, not the Open edX guide. 
My thought was, while I can speak with some certainty about what learners will find in edx.org courses (which are by definition MOOCs), it gets a lot fuzzier for non-edx.org site. 
What do you think?

File: en_us/shared/students/SFD_introduction.rst [hunk 16]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  course teams are continuously looking for new ways to engage learners and
+  teach more effectively. The course teams typically include instructions for
+  how to use any enhanced components right in the course.
+
+  MOOCs bring learners together from all over the world. You have a wide
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: "You have" seems like it should be "you bring" so that it works with the end of the sentence. But there's already a "bring" in the first sentence. 

MOOCs gather learners from all over the world. You bring a wide variety...

File: en_us/shared/students/SFD_introduction.rst [hunk 17]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+  variety of educational backgrounds, and educational goals, to the edX courses
+  that you take. Regardless of your background or your reasons for taking an
+  edX course, a part of the MOOC learning experience that everyone shares is
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: strike "that everyone shares."

File: en_us/shared/students/SFD_introduction.rst (hunk 14 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: crud, now I realize that this entire section (not just my new bit) is written as though for edX only, but it's included in the Open edX learners guide...

File: en_us/shared/students/SFD_introduction.rst [hunk 18]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+
+  If you have a question about something you encounter in an edX course, try
+  these options for getting an answer.
+
+  * Check the **Home** page. Course teams use this page to post updates about
+    the course, which can include reminders about when graded assignments are
+    due, or announce upcoming events or milestones.
+
+  * Participate in the :ref:`course discussions<Explore Posts>`. If you
+    encounter an unfamiliar word or phrase, such as "finger exercise" or "board
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: I think I understand why you included the specific terms that came up the other day. Does showing those terms imply that learners should know what they mean? Maybe reword the example so that it's more explicit about the fact that a course team used a term that learners didn't understand?

File: en_us/shared/students/SFD_introduction.rst [hunk 19]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    work", search for it on the **Discussion** page, or post a question about
+    it yourself. Other learners might be able to answer your question, or might
+    have the same question themselves. Your comments and questions give the
+    course team useful feedback for improving the course.
+
+  * Investigate other resources. Within your course, a course
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: Strike "Within your course."

File: en_us/shared/students/SFD_introduction.rst (hunk 14 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: ok, I'm going to take a step back and rewrite as though for any learner on edx.org OR Open edX...

Comment (lamagnifica) [inline en_us/shared/students/SFD_introduction.rst]: grr arrgh

File: en_us/shared/students/SFD_introduction.rst [hunk 20]
```diff
@@ -8,13 +8,66 @@ Welcome to edX! We're glad you're taking a course with us and we hope you're as
...
+    :ref:`wiki<SFD Wiki>`, if one is available, can be a good source of
+    information. The course team might also have a course-specific Facebook
+    page or Twitter feed set up for learners to share questions and answers.
```
Comment (pdesjardins) [inline en_us/shared/students/SFD_introduction.rst]: Facebook® page or Twitter® feed...

Too much? I actually don't like assuming that everyone has heard of those products. Even though everyone has.

**Comment by pdesjardins** [comment]
Great content! I finished my review, comments added. 👍

File: en_us/shared/students/SFD_introduction.rst [hunk 21]
```diff
@@ -4,29 +4,87 @@
...
+background or your reasons for taking an online course, one part of the
+learning experience that everyone shares is encountering new ways of learning.
+As a result, most learners occasionally have questions about what is expected,
+how to complete an assignment, or what a word, phrase, or abbreviation means.
+
+If you have a question about something you encounter in an edX course, try
+these options for getting an answer.
+
+* Check the **Home** page. Course teams use this page to post updates about the
+  course, which can include reminders about when graded assignments are due, or
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: @lamagnifica I still think it's worthwhile to mention something in your "which can include" which ties more closely back to the "if you have a question about something you encounter" - otherwise this introduces a bit of a disconnect, especially the first bullet after your intro sentence.
"...which can include explanations about course content, reminders about due dates for graded assignments, or announcements about upcoming events or deadlines."

File: en_us/shared/students/SFD_introduction.rst [hunk 22]
```diff
@@ -4,29 +4,87 @@
...
+and practice using them, by enrolling in the `edX Demo course`_.
+
+Depending on the course, a variety of enhanced components might also be
+included, from team problem-solving exercises and lab work to live
+demonstrations and ask me anything (AMA) sessions. Every course has a unique
+design, and course teams are continuously looking for new ways to engage
+learners and teach more effectively. Course teams typically include
+instructions for how to use any enhanced components right in the course.
+
+MOOCs bring learners together from all over the world. The learners have a wide
```
Comment (catong) [inline en_us/shared/students/SFD_introduction.rst]: Talking about "the learners" as if the reader isn't one is distracting. How about: "MOOCs bring together learners from all over the world who have a wide variety of educational backgrounds and educational goals. Regardless of your..."

**Comment by catong** [comment]
@lamagnifica This is a great addition. Made a couple more editorial suggestions that still jumped out at me on last reading. Otherwise 👍
